scheduler/pkg/klcpermit/permit.go

```diff
@@ -31,16 +31,16 @@ func (pl *Permit) Name() string {
 
 func (pl *Permit) Permit(ctx context.Context, state *framework.CycleState, p *v1.Pod, nodeName string) (*framework.Status, time.Duration) {
 
-	klog.Infof("[Keptn Permit Plugin] waiting for pre-deployment checks on %s", p.GetObjectMeta().GetName())
+	klog.Infof("[Keptn Permit Plugin] waiting for pre-deployment checks on %s", p.GetObjectMeta().GetName(), p.GetObjectMeta().GetNamespace())
```

Comment:
I it expected to use only one `%s` in the log message and fetch two information from the object (name, namespace) ?

I would suggest to have a specific `%s` for the name and also for namespace of the object.

Reply:
Done. made the fix